---
title: "scMinerva: an Unsupervised Graph Learning Framework with Label-efficient Fine-tuning for Single-cell Multi-omics Integrative Analysis"
collection: publications
permalink: /publications/01-2022-05-29-scMinerva
excerpt: 'Single-cell multi-omics is a rapidly growing field in biomedicine, where multiple biological contents, such as the epigenome, genome, and transcriptome, can be measured simultaneously. Despite its potential, the integrated analysis and prediction of cellular states based on this complex multi-omics data pose significant challenges due to data sparsity, high noise, and computational overhead. To address these challenges, we developed scMinerva, an unsupervised framework for single-cell multi-omics integrated analysis. The learned embeddings from the multi-omics data enable accurate integrated classification of cell types and stages. Specifically, we construct a heterogeneous graph from multiple omics and propose a novel biased random walk algorithm omics2vec, which can learn the heterogeneous biological graph in a way that balances both local and global network structures. scMinerva successfully outperforms existing unsupervised methods on various simulated and real-world datasets when fine-tuned by very few labels.'
date: 2022-05-29
venue: 'biorXiv'
authors: 'Tingyang Yu, Yongshuo Zong, Yixuan Wang, Xuesong Wang, Yu Li'
paperurl: 'https://yistyu.github.io/files/scMinerva.pdf'
citation: 'Yu T, Zong Y, Wang Y, Wang X & Li Y. scMinerva: a GCN-featured Interpretable Framework for Single-cell Multi-omics Integration with Random Walk on Heterogeneous Graph[J]. bioRxiv (Submitted to Nature Communications), 2022.'
---

**Tingyang Yu**, Yongshuo Zong, Yixuan Wang, Xuesong Wang, Yu Li

Single-cell multi-omics is a rapidly growing field in biomedicine, where multiple biological contents, such as the epigenome, genome, and transcriptome, can be measured simultaneously. Despite its potential, the integrated analysis and prediction of cellular states based on this complex multi-omics data pose significant challenges due to data sparsity, high noise, and computational overhead. To address these challenges, we developed scMinerva, an unsupervised framework for single-cell multi-omics integrated analysis. The learned embeddings from the multi-omics data enable accurate integrated classification of cell types and stages. Specifically, we construct a heterogeneous graph from multiple omics and propose a novel biased random walk algorithm omics2vec, which can learn the heterogeneous biological graph in a way that balances both local and global network structures. scMinerva successfully outperforms existing unsupervised methods on various simulated and real-world datasets when fine-tuned by very few labels. Additionally, scMinerva demonstrates strong label efficiency, is robust to fluctuation in data quality, allows one omics to compensate for weakness in others and could effectively classify cells with different annotation granularities. Furthermore, we showcase scMinerva's ability to accurately provide prospective biomarkers and predict cell differentiation trends for COVID-19-infected cells, through the joint analysis of multi-omics data.

[Download paper here](https://yistyu.github.io/files/scMinerva.pdf)
