---
title: "Contrastive Cycle Adversarial Autoencoders for Single-cell Multi-omics Alignment and Integration"
collection: publications
permalink: /publications/04-2021-12-05-cae
excerpt: 'Muilti-modality data are ubiquitous in biology, especially that we have entered the multi-omics era, when we can measure the same biological object (cell) from different aspects (omics) to provide a more comprehensive insight into the cellular system. When dealing with such multi-omics data, the first step is to determine the correspondence among different modalities. In other words, we should match data from different spaces corresponding to the same object. To promote the single-cell multi-omics research, we proposing a novel framework to align and integrate single-cell RNA-seq data and single-cell ATAC-seq data. Our approach can efficiently map the above data with high sparsity and noise from different spaces to a low-dimensional manifold in a unified space, making the downstream alignment and integration straightforward. Compared with the other state-of-the-art methods, our method performs better in both simulated and real single-cell data. The proposed method is helpful for the single-cell multi-omics research. The improvement for integration on the simulated data is significant.'
date: 2021-12-05
venue: 'arXiv (Major revision at Bioinformatics)'
authors: 'Xuesong Wang, Zhihang Hu, Tingyang Yu, Ruijie Wang, Yumeng Wei, Juan Shu, Jianzhu Ma, Yu Li'
paperurl: 'https://scholar.google.com/citations?view_op=view_citation&hl=zh-CN&user=1Cw8oZ4AAAAJ&citation_for_view=1Cw8oZ4AAAAJ:u5HHmVD_uO8C'
citation: 'Wang, X., Hu, Z., Yu, T., Wang, R., Wei, Y., Shu, J., ... & Li, Y. (2021). Contrastive cycle adversarial autoencoders for single-cell multi-omics alignment and integration (Major revision at Bioinformatics (IF = 6.93)). arXiv preprint arXiv:2112.03266.'
---

Xuesong Wang, Zhihang Hu, **Tingyang Yu**, Ruijie Wang, Yumeng Wei, Juan Shu, Jianzhu Ma, Yu Li

Muilti-modality data are ubiquitous in biology, especially that we have entered the multi-omics era, when we can measure the same biological object (cell) from different aspects (omics) to provide a more comprehensive insight into the cellular system. When dealing with such multi-omics data, the first step is to determine the correspondence among different modalities. In other words, we should match data from different spaces corresponding to the same object. This problem is particularly challenging in the single-cell multi-omics scenario because such data are very sparse with extremely high dimensions. Secondly, matched single-cell multi-omics data are rare and hard to collect. Furthermore, due to the limitations of the experimental environment, the data are usually highly noisy. To promote the single-cell multi-omics research, we overcome the above challenges, proposing a novel framework to align and integrate single-cell RNA-seq data and single-cell ATAC-seq data. Our approach can efficiently map the above data with high sparsity and noise from different spaces to a low-dimensional manifold in a unified space, making the downstream alignment and integration straightforward. Compared with the other state-of-the-art methods, our method performs better in both simulated and real single-cell data. The proposed method is helpful for the single-cell multi-omics research. The improvement for integration on the simulated data is significant.

[Download paper here](https://arxiv.org/pdf/2112.03266.pdf)
